client/src/pages/dashboard/components/Bank/FavoriteCurrency/BasicCurrency.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import Select from "react-select";
+import { FormGroup, Button, TabPane, Row, Col } from "reactstrap";
+import makeAnimated from "react-select/animated";
+import { any, func } from "prop-types";
+import { Head6 } from "./style";
+
+const BasicCurrency = ({ toggle, options, chooseCurrency }) => {
+  return (
+    <TabPane tabId="1">
+      <Row>
+        <Col sm="12">
+          <FormGroup>
+            <Head6> Please choose basic currency</Head6>
+            <Select
+              closeMenuOnSelect={false}
+              components={makeAnimated()}
+              options={options}
+              onChange={chooseCurrency}
+            />
+          </FormGroup>
+          <Button
+            color="primary"
+            onClick={() => {
+              toggle("2");
+            }}
```

Comment:
Don't create new function in attribute onClick, please. Create a separate variable for that.

Reply:
https://github.com/dimapanasiuk/mern/commit/13c28c19f8fb2955c69f0219174dd2716cf94c46